content/en/docs/contribute/new-content/new-features.md

```diff
@@ -0,0 +1,139 @@
+directory. The sub-directory contains a release schedule, a draft of the release
+notes, and a document listing each person on the release team.
+
+The release schedule contains links to all other documents, meetings,
+meeting minutes, and milestones relating to the release. It also contains
+information about the goals and timeline of the release, and any special
+processes in place for this release. Near the bottom of the document, several
+release-related terms are defined.
+
+This document also contains a link to the **Feature tracking sheet**, which is
+the official way to find out about all new features scheduled to go into the
+release.
+
+The release team document lists who is responsible for each release role. If
+it's not clear who to talk to about a specific feature or question you have,
+either attend the release meeting to ask your question, or contact the release
+lead so that they can redirect you.
+
+The release notes draft is a good place to find out a little more about
+specific features, changes, deprecations, and more about the release. The
+content is not finalized until late in the release cycle, so use caution.
+
+### Feature tracking sheet
+
+The feature tracking sheet
+[for a given Kubernetes release](https://github.com/kubernetes/sig-release/tree/master/releases) lists each feature that is planned for a release.
+Each line item includes the name of the feature, a link to the feature's main
+GitHub issue, its stability level (Alpha, Beta, or Stable), the SIG and
+individual responsible for implementing it, whether it
+needs docs, a draft release note for the feature, and whether it has been
+merged. Keep the following in mind:
+
+- Beta and Stable features are generally a higher documentation priority than
+  Alpha features.
+- It's hard to test (and therefore, document) a feature that hasn't been merged,
+  or is at least considered feature-complete in its PR.
+- Determining whether a feature needs documentation is a manual process and
+  just because a feature is not marked as needing docs doesn't mean it doesn't
+  need them.
+
+## For developers or other SIG members
+
+This section is information for members of other Kubernetes SIGs documenting new features
+for a release.
+
+If you are a member of a SIG developing a new feature for Kubernetes, you need
+to work with SIG Docs to be sure your feature is documented in time for the
+release. Check the
+[feature tracking spreadsheet](https://github.com/kubernetes/sig-release/tree/master/releases)
+or check in the `#sig-release` Kubernetes Slack channel to verify scheduling details and
+deadlines.
+
+### Open a placeholder PR
+
+1. Open a pull request against the
+`release-X.Y` branch in the `kubernetes/website` repository, with a small
+commit that you will amend later.
+2. Use the Prow command `/milestone X.Y` to
+assign the PR to the relevant milestone. This alerts the docs person managing
```

Comment:
Check whether a developer has the privileges to set a milestone

Reply:
Hm, good catch. I need more info on who winds up in the milestone-maintainers GH group – @zacharysarah?